CHANGES_NEXT_RELEASE

```diff
@@ -1,2 +1,3 @@
+- fix: check data access in function cbDoReqUpdAxn (#659)

```

Comment:
Maybe more user oriented this other alternative?

```suggestion
- Fix: improve CB error management when response has no data (#659)
```

Not sure if the error is from CB or Keystone... please adjust if my suggestion is not correct.